src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeOptionsFormDataProvider.php

```diff
@@ -60,6 +71,33 @@ public function getData()
      */
     public function setData(array $data)
     {
+        if ($errors = $this->validate($data)) {
+            return $errors;
+        }
+
         return $this->employeeOptionsConfiguration->updateConfiguration($data);
     }
+
+    /**
+     * Perform validation on form data before saving it.
+     *
+     * @param array $data
+     *
+     * @return array Returns array of errors
+     */
+    protected function validate(array $data)
+    {
+        $errors = [];
+
+        $passwordChangeTime = $data['password_change_time'];
+        if (!is_numeric($passwordChangeTime) || 0 > $passwordChangeTime) {
+            $errors[] = [
```

Comment:
it might not be the scope of PR, but I don't like this "free array" return type, especially if we already use such in multiple places.
It deserves a dedicated data transfer object, so we can type it and avoid the mess of dealing with array keys.